src/constants.py

```diff
@@ -15,6 +15,9 @@
 LOG_DIR = PurePath("/var/log")
 LOG_FILE = LOG_DIR / "admin_ui.log"
 RULES_CONFIGMAP_FILE_NAME = "admin_ui_rules.json"
+OAUTH_SCOPES = "openid profile offline_access"
+OAUTH_GRANT_TYPES = ["authorization_code", "refresh_token"]
+OAUTH_CALLBACK_PATH = "api/v0/login"
```

Comment:
FYI: with my next PR logic is split between two handlers (one for login init, the second one for completing the flow), and the callback will actually be `/api/v0/auth/callback`